src/components/challengeWidget/challengeWidget.js

```diff
@@ -0,0 +1,40 @@
+//import logo from './logo.svg';
```

Comment:
This component must be exported here and imported in components so it can be used in the app